### ISSUE TITLE
Datalist on HTML input does not expand when clicked

### ISSUE THREAD

**Issue by mrudelle** [electron/electron#14879]
* Output of `node_modules/.bin/electron --version`: v3.0.2
* Operating System (Platform and Version): OSX 10.13.6
* Chromium version: 66.0.3359.181

**Expected Behavior**
When I add an html datalist to an input the list of elements in the list should appear when i click on the arrow button.

The list is added as such: (see forked repo in how to reproduce section)
```html
<input list="framerate-list" type="text">
<datalist id="framerate-list">
    <option value="24">24fps</option>
    <option value="25">25fps</option>
    <option value="30">30fps</option>
    <option value="60">60fps</option>
</datalist>
```

Expected (in chrome):
<img width="509" alt="screen shot 2018-09-29 at 13 47 14" src="https://user-images.githubusercontent.com/3630020/46245472-4f028c80-c3ee-11e8-97be-c0dc441efcad.png">

**Actual behavior**
The arrows appear as if the list can be shown. But nothing appear when the arrow is clicked.
Note: When I run `$ open index.html` the list appears normally in chrome.

In electron:
<img width="665" alt="screen shot 2018-09-29 at 13 45 49" src="https://user-images.githubusercontent.com/3630020/46245448-0d71e180-c3ee-11e8-84b6-ebf9abf91dd4.png">

**To Reproduce**
```sh
$ git clone https://github.com/mrudelle/electron-quick-start.git
$ npm install
$ npm run
```

**Comment by PixleProgrambetaxl** [comment]
+1

**Comment by avsmithy** [comment]
Hi, I'm running into this with v4.0.4, any updates or suggestions?

**Comment by brenca** [comment]
This should be fixed in 5.0.x, see #360 for more info.

**Comment by mrudelle** [comment]
Awesome! Thanks @brenca for the fix, that will be very useful!

Not yet fixed in 5.0.0-beta.3 though, might be because this backport is still pending: https://github.com/electron/electron/pull/16880 or it's just my general comprehension of how the different branches work ^^ will give it some time and try again.

**Comment by avsmithy** [comment]
+1 Thank you @brenca

**Comment by brenca** [comment]
@mrudelle yeah, once that backport PR is merged, it should be in the next beta.

**Comment by laurent22** [comment]
We would really need this fix as well in our application. Will it be backported to Electron 4 too?

**Comment by Devhercule** [comment]
Not yet fixed in 5.0.1 version :(
